src/apps/tabs/components/default-tab.react.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Dependencies.
+ */
+
+let React = require('react');
+let InfiniteScroll = require('react-infinite-scroll')(React);
+let moment = require('moment');
+let cache = require('lscache');
+let ProductGroup = require('./product-group.react');
+let Header = require('./header.react');
+let ProductStore = require('../../../common/stores/product');
+let api = require('../../../common/api');
+let getDay = require('../util/get-day');
+let groupByDay = require('../util/group-by-day');
+let Pane = require('../../../common/product-pane/pane.react');
+
+/**
+ * Constants.
+ */
+
+const CACHE_KEY = process.env.PRODUCTS_CACHE_KEY;
```

Comment:
One problem with the `env` we have even in the web app is the `process.env` is used in a lot of places. I think it would make sense to have a `config` module which gets all its configs from `process.env` it can be like:

```js
config.get('products_cache_key`) # => if key is not set it will raise an error
```